nss_cache/sources/ldapsource.py

```diff
@@ -519,10 +520,15 @@ def GetUpdates(self, source, search_base, search_filter,
 class PasswdUpdateGetter(UpdateGetter):
   """Get passwd updates."""
 
-  def __init__(self):
+  def __init__(self, conf):
     super(PasswdUpdateGetter, self).__init__()
```

Comment:
i.e. super(..).__init__(conf)

and in line 423 add a new __init__ method that accepts conf and move line 525 below into this new method.